meteor-app/server/api/v1/reports/methods.ts

```diff
@@ -13,6 +15,8 @@ Meteor.methods({
 		TODO: restrict to account and user roles
 	*/
 	Insert_Report_Data: function(json) {
+		enforceRole(this.userId, 'Editor')
+		check(json, Match.ObjectIncluding({collection_name : String}))
 		Report_Data.insert({accountId: 'fyS84mmYeNLqDuaSS', ...json})
```

Comment:
Lets complete the TODO here, and use the correct account ID, rather than the hard coded one